style.css

```diff
@@ -1614,4 +1605,14 @@ input#Email {
     width: 200px;
     height: 40px;
   }
+
+  .lineD {
+    margin-top: 698px;
+    width: 100%;
+  }
+
+  .subInput :nth-child(4) {
+    display: flex;
+    justify-content: flex-start;
+  }
 }
```

Comment:
Kindly implement these changes for different sections of your project to improve the UI of your project

**NAVBAR SECTION**

-  Please adjust the margin of the nav items so it is away from the border line just as it is in the design given 😄 

Figma design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/a8b91fcd-f5da-4f7f-be2d-0e93ee6d3236)

Your design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/065e3e38-fb4f-4a77-a5eb-8ef236c126c1)

**PROJECT SESSION**

- Kindly use the right color for the heading `My recent works` so it is similar to the design given 

Figma design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/7b2a42b6-8442-488c-82fc-6be395724c2a)

Your design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/e8956358-33a2-4628-b2fd-44e2366e3a0d)

-  Kindly add some padding in the technolies of your project section
 Figma design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/821e858e-4de6-4753-bca5-496d217fbe5d)

Your design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/40d7d99a-65aa-424d-9b13-1121b789d3ba)

**ABOUT ME **

-  Kindly rescue the margin-bottom here, its really big and can make viewing unpleasurable , please it should be similar to the figma template. Also, use the right font-type and font properties for your project , especially in this section 
-  Please use the right background-color for this section. Your design background-color is darker than what we have in the design 😄 

your design :
![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/42255671-177e-4687-a193-ccc15a4241b1)

**CONTACT SECTION**

- Kindly change the background-color for this text in your design :

![image](https://github.com/bhushan354/My-Portfolio/assets/20537712/764fa7ad-3e81-4d63-9952-9ff60dc7833f)